docs/configuration-reference.md

```diff
@@ -1812,6 +1812,25 @@ basic_auth:
 # read from the configured file. It is mutually exclusive with `bearer_token`.
 [ bearer_token_file: /path/to/bearer/token/file ]
 
+# Configures SigV4 request signing. The default credentials chain will be used,
+# documented here:
+#
+# https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials
+#
+# When enabled, region must be supplied.
+#
+# This feature is currently exclusive to the Grafana Cloud Agent and is only
+# currently available for remote_write of Prometheus metrics.
+sigv4:
+  # Enable SigV4 request signing. May not be enabled at the same time as
+  # configuring basic auth or bearer_token/bearer_token_file.
+  [ enabled: <boolean> | default = false ]
+
+  # Region to use for signing the requests. When sigv4.enabled is true,
```

Comment:
Can we use the default region if nothing here? Might not be something we do now, but in the aws config you can specify the region.

Reply:
I think my concern is the AWS config might have a different region than the remote_write URL. WDYT?

Reply:
I think given the time constraint I'd prefer making this required for now and follow up with some kind of pleasant defaulting later